bands/templates/band_admin/add_band.html

```diff
@@ -0,0 +1,75 @@
+{% extends "base.html" %}{% set admin_area=True %}
+{% block title %}Group Form{% endblock %}
+{% block content %}
+
+<style>
```

Comment:
I'm okay with having a `<style>` tag on a specific page if it is only relevant to that page. In this instance, it's not 100% specific to this page :(

This is another instance where I didn't want to open a separate pull request in the main repo just to add styling for this one page :(